src/ipc-handlers.ts

```diff
@@ -161,8 +167,7 @@ export async function createSite(
 	customDomain?: string,
 	enableHttps?: boolean,
 	siteId?: string
-): Promise< SiteDetails[] > {
-	const userData = await loadUserData();
+) {
```

Comment:
```suggestion
): Promise< SiteDetails > {
```
Per my other comment about return types.